docs_devel/HowToCreateFilterPlugin.md

```diff
@@ -23,6 +23,8 @@ OmegaT file filter plugin.
 There must be a manifest file that indicates that it is an OmegaT plugin. There are two flavors, see below. 
 Omegat 5.3.0 also supports to provide additional information (valid for both flavors) that can be displayed in the UI. 
 You can **optionally** provide name, version, author and description. 
+OmegaT 5.5.0 show plugin name, and author in preference dialog. You are recommended to set these parameters.
+You can **optionally** provide link URL of your plugin home page, License and category.
```

Comment:
>  You can **optionally** provide link URL of your plugin home page, License and category.

This is not yet implemented in `master`, right?

Reply:
yes, discussed in #92